code/code/spec/spec_objs.cc

```diff
@@ -6785,7 +6785,7 @@ extern int livingVines(TBeing *vict, cmdTypeT cmd, const char *, TObj *o, TObj *
 extern int dkSword(TBeing *vict, cmdTypeT cmd, const char *, TObj *o, TObj *);
 extern int moltenWeapon(TBeing *vict, cmdTypeT cmd, const char *arg, TObj *o, TObj *);
 extern int glacialWeapon(TBeing *vict, cmdTypeT cmd, const char *arg, TObj *o, TObj *);
-
+extern int lightningboltWeapon(TBeing *vict, cmdTypeT cmd, const char *arg, TOb;
```

Comment:
Whoa, you're now mixing up egotrip and lightning bolt. That's no bueno.

Reply:
See below